TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -289,6 +289,15 @@ public static void OnCodeElement(StartStatement statement, CodeElementsParser.St
         }
     }
 
+    class StopStatementChecker
+    {
+        public static void OnCodeElement(StopStatement statement, CodeElementsParser.StopStatementContext context)
+        {
+            if (statement.StopRun != null && statement.StopRun.Value)
+                DiagnosticUtils.AddError(statement, "GOBACK should be used instead of STOP RUN", context, MessageCode.Warning);
```

Comment:
There is no need to display the RuleStack for this error.
So the parameter "context" can be removed.

Reply:
done

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/08ca1ff0a45ef945f24534bde9a3d5535cf5d2ab